src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$white-accent: #fff;
+$blue-accent: #00acdc;
```

Comment:
You need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.